catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Remove whitespace and literal newlines
+     * @param input
+     * @return
+     */
+    private String sanitize(final String input) {
```

Comment:
you can assert that the input is the same as the output, ie
string1 = "abcdefg"
assertThat(string1, is(sanitize(string1))
string2 = "abc def g"
assertThat(string2, is(sanitize(string2))
string3 = "abc def  g"
assertThat(string3, is(not(sanitize(string3)))